scripts/fill_virtualtopic_names.py

```
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+"""Rellena nombre y descripción en VirtualTopic existentes."""
+import sys
+import os
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from datetime import datetime
+from bson import ObjectId
+from src.shared.database import get_db
+
+def main():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)